styles/styles.css

```diff
@@ -20,20 +23,312 @@
+    position: relative;
+}
+
+.header-navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+}
+
+.header-navigation a {
+    color: #FFF;
+}
+
+.header-navigation ul {
+    flex-basis: 400px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.header-text {
+    font-size: 28px;
+    padding: 100px 0;
+    text-align: center;
+    max-width: 850px;
+    font-weight: 300;
+}
+
+.header img {
+    position: absolute;
+    max-width: 80px;
+    top: 2rem;
+    margin: 360px 0 0 0;
+    padding: 210px 0;
+}
+
+
 /* --------------- 4.2 Introduction ------------------- */
 
+#introduction {
+    padding: 100px;
+}
+
+#introduction h3 {
+    text-align: center;
+    max-width: 850px;
+}
+
+#introduction p {
+    padding: 100px 0 0 0;
+    font-weight: 400;
+}
+
+/* --------------- 4.3 Projects ------------------- */
+
+#projects {
+    background-color: var(--light-grey);
+    font-weight: 400;
+    padding: 100px 50px;
+}
+
+#projects h3 {
+    color: var(--black);
+}
+
+#projects a {
+    text-transform: uppercase;
+    font-weight: 600;
+    font-size: 15px;
+    letter-spacing: 2px;
+}
+
+.article-projects {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 0 0 100px 0;
+    align-items: center;
+}
+
+.article-projects:nth-child(even) {
+    flex-direction: row-reverse;
+}
+
+.article-projects-content {
+    padding: 50px 50px;
+    flex-basis: 500px;
+    flex-grow: 0;
+}
+
+.article-projects p {
+    padding: 40px 0;
+}
+
+.image-wrapper {
+    flex-basis: 800px;
+    flex-shrink: 0;
+}
+
+.image-wrapper img {
+    display: block;
+    width: 100%;
+    flex-shrink: 0;
+    flex-grow: 0;
+}
+
+.projects-squiggle img {
+    max-width: 55px;
+    align-content: flex-start;
+}
+
+.article-projects span {
+    box-shadow: 0 10px 10px rgba(0, 0, 0, 0.20);
+}
+
+/* --------------- 4.4 Services ------------------- */
+
+#services p {
+    font-size: 16px;
+    margin-top: 20px;
+}
+
+.services-tile img {
+    max-width: 80px;
+    margin: 10px 0;
+}
+
+.services-tile {
+    border: 1px solid var(--purple);
+    flex-basis: 380px;
+    flex-wrap: wrap;
+    padding: 40px;
+    box-shadow: 0 10px 10px rgba(0, 0, 0, 0.20);
+}
+
+.inner-container__services-tile {
+    flex-basis: 1400px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    row-gap: 40px;
+    margin: 0 20px;
+    padding: 100px;
+}
+
+/* --------------- 4.5 Contact ------------------- */
+
+#contact {
+    background: rgb(72, 69, 228);
+    background: linear-gradient(150deg, rgba(72, 69, 228, 1) 0%, rgba(87, 34, 205, 1) 100%);
+    color: #FFF;
+    position: relative;
+    padding: 100px 0 40px;
+}
+
+#contact form {
+    display: flex;
+    flex-direction: column;
+    width: 850px;
+}
+
+#contact button {
+    width: 200px;
```

Comment:
![image](https://github.com/user-attachments/assets/a2a5872a-2d44-41d7-856e-af7fc617f310)

Wat als iemand er ineens 'Neem heel snel contact met ons op door op deze button te klikken' in zet?